pkgs/top-level/all-packages.nix

```diff
@@ -38870,6 +38870,8 @@ with pkgs;
     jre = jre8; # TODO: remove override https://github.com/NixOS/nixpkgs/pull/89731
   });
 
+  owmods-cli = callPackage ../application/misc/owmods-cli { };
```

Comment:
```suggestion
  owmods-cli = callPackage ../applications/misc/owmods-cli { };
```

Reply:
Fixed the mistake with `application` and cleaned the commit message.